dashboard/app/models/script.rb

```diff
@@ -771,7 +771,7 @@ def self.unit_names_by_curriculum_umbrella(curriculum_umbrella)
   def self.units_with_standards
     Script.
       where("properties -> '$.curriculum_umbrella' = 'CSF'").
-      where("properties -> '$.version_year' >= '2019'").
+      where("properties -> '$.version_year' >= '2019' and properties -> '$.version_year' < '#{CourseVersion::UNVERSIONED}'").
```

Comment:
what is the mental model here -- does blank or empty mean one thing and "unversioned" means another?

Reply:
Yeah that's a good question. Blank seems to have historically meant "unversioned" but that wouldn't allow us to add a course version to those scripts, so this is kind of a placeholder. I think in a lot of ways blank and "unversioned" mean the same thing but ideally at some point "unversioned" scripts will have more options available, such as resources, vocab, and potentially more published_state control (tbd on that).

Reply:
> Blank seems to have historically meant "unversioned" but that wouldn't allow us to add a course version to those scripts

this is because the course_version.key field cannot be blank, right? https://github.com/code-dot-org/code-dot-org/blob/f4261cfbfc7251379c0919440247d883662549d6/dashboard/app/models/course_version.rb#L6

I think it is fine to preserve this requirement and go with "unversioned" here. my request would be to add a comment explaining the significance of blank and "unversioned".

Reply:
Yeah, we have to add a key in course_version and I'm not sure any way around that. I don't have any strong preference for what we have the key be so open to suggestions here.

I'll add some comments